plugin/rooter.vim

```diff
@@ -192,8 +192,8 @@ command! Rooter :call <SID>ChangeToRootDirectory()
 if !exists('g:rooter_manual_only') || !g:rooter_manual_only
   augroup rooter
     autocmd!
-    autocmd VimEnter,BufEnter * :Rooter
-    autocmd BufWritePost * :call setbufvar('%', 'rootDir', '') | :Rooter
+    autocmd VimEnter,BufEnter * nested :Rooter
+    autocmd BufWritePost * nested :call setbufvar('%', 'rootDir', '') | :Rooter
```

Comment:
I don't think you need `nested` here because all it does is set a buffer variable.

Reply:
You mean line 196? I simply added it as it eventually calls Rooter, just to be safe. It certainly works with only Line 195.

Want me to redo the commit?

Reply:
Ah, you're right; somehow I didn't notice the `| :Rooter` on the end of line 196.  Let's leave it just to be safe.